src/http/http.js

```diff
@@ -82,9 +84,11 @@ function makeRequest (endpoint, options) {
     ...rest
   } = options
   // Build fetch config
+  const allHeaders = { ...DEFAULT_OPTIONS.headers, ...getAuthHeaders(bearerToken), ...headers }
+  const requestHeaders = overrideHeaders ? { ...headers } : allHeaders
```

Comment:
You can just use `headers` instead of `{ ...headers }`

Reply:
Haha, oops!